alchemist-loading/src/main/kotlin/it/unibo/alchemist/boundary/loader/VariablesOverrider.kt

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2010-2023, Danilo Pianini and contributors
+ * listed, for each module, in the respective subproject's build.gradle.kts file.
+ *
+ * This file is part of Alchemist, and is distributed under the terms of the
+ * GNU General Public License, with a linking exception,
+ * as described in the file LICENSE in the Alchemist distribution's top directory.
+ */
+
+package it.unibo.alchemist.boundary.loader
+
+import it.unibo.alchemist.boundary.modelproviders.YamlProvider
+
+/**
+ * Use this to override a map of variables with a list of resolvable key-value pairs.
+ */
+object VariablesOverrider {
```

Comment:
Probably better to see this 

```suggestion
internal object Overrides {
```

Reply:
Done